docs/reference/search/search-shards.asciidoc

```diff
@@ -165,4 +165,4 @@ routing values have been specified.
 `local`::
     A boolean value whether to read the cluster state locally in order to
     determine where shards are allocated instead of using the Master node's
-    cluster state.
+    cluster state.
```

Comment:
can you update the sentence: "When filtered aliases are used, the filter is returned as part of the indices section [5.1.0] Added in 5.1.0." ?